ballerina/Module.md

```diff
@@ -1,10 +1,10 @@
 ## Overview
 
-This module provides the functionality required to access and manipulate data stored in an MSSQL database.
+This package provides the functionality required to access and manipulate data stored in an MSSQL database.
```

Comment:
```suggestion
This module provides the functionality required to access and manipulate data stored in an MSSQL database.
```

Reply:
@kaneeldias we don't need to change it to package in `module.md`. Only in package.md we call it package. because from a Central point of view, it is a package. Shall we revert this change

Reply:
In Module.md -> Its `module`
Package.md -> `package`
README -> `library`

Reply:
Fixed in  b6d4892